foxpuppet/windows/browser/notifications/addons.py

```diff
@@ -33,8 +33,8 @@ def addon_name(self):
         """
         with self.selenium.context(self.selenium.CONTEXT_CHROME):
             if self.window.firefox_version >= 55:
-                return self.root.find_anonymous_element_by_attribute(
-                    'anonid', 'description').find_element(
+                return self.root.find_element(
+                    By.ID, 'addon-webext-perm-header').find_element(
```

Comment:
Can you take care of this in a separate pull request? It's not related to the navigation bar or tracking shield.